README.md

```diff
@@ -12,28 +23,16 @@
 |Nomad|0.12.3 or newer|0.12.3 or newer|
 
 ## Usage
-
-### Starting datastack
-Run
-```makefile
-make up
-```
-in your terminal to provision the datastack. When the process is finished you can then open up access to Redash, Presto, and MinIO by running
-```makefile
-make connect-to-all
-```
-which will open 3 terminal windows each with a connection to one of the components. Alternatively you can run `make proxy-redash`, `make proxy-presto`, and `make proxy-minio` to open individual proxy connections.
-
 ### Requirements
 #### Required modules
 - [MinIO](https://github.com/fredrikhgrelland/terraform-nomad-minio)
 - [Hive](https://github.com/fredrikhgrelland/terraform-nomad-hive)
```

Comment:
Are these modules  required for redash module in general or they are currently only required when one runs the example/datastack . I am not sure if we should have these here.

Reply:
Good point. They are not required, and this header should be reserved to for example the postgres module.

Reply:
I've removed them now